.travis.yml

```diff
@@ -7,17 +7,30 @@ php:
     - 5.6
     - hhvm
 
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
```

Comment:
`$HOME/.composer/cache` is the directory where the dist files are cached by Composer (that's what used when you see `loading from cache`). By caching this directory, Composer will not download the dist files each test, but only the new dist files (and caches then directly).

We could also cache `./vendor`, but Composer has a bug in how it deals with branch aliases, making it a bad choice to do this.

About the new environment and caching on Travis: http://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/